src/environment.jl

```diff
@@ -287,7 +287,7 @@ It is unaffected by [`MPI.Finalize`](@ref), and is one of the few functions that
 called before [`MPI.Init`](@ref).
 
 # External links
-$(_doc_external("MPI_Intialized"))
+$(_doc_external("MPI_Initialized"))
```

Comment:
Ugh, maybe we should have a check for the documentation that links aren't broken 😕

Reply:
Is there a option in Documenter.jl for that?

Reply:
Apparently yes: #856.  But MPICH/OpenMPI websites aren't reliable for actually documenting all functions... So the reality is that we do have a large number of broken links 😞